pybossa/view/projects.py

```diff
@@ -3180,7 +3180,7 @@ def integrate_ext_config(config_dict):
             cf = {}
             for field in content.get('fields') or {}:
                 name = field['name']
-                if config_dict[name]:
+                if config_dict.get(name) and config_dict[name]:
```

Comment:
is `if config_dict.get(name):` enough?

Reply:
this will ensure that config_dict has key and value in it.